paasta_tools/mesos_tools.py

```diff
@@ -139,8 +139,8 @@ def filter_not_running_tasks(tasks):
     return [task for task in tasks if not is_task_running(task)]
 
 
-def get_running_tasks_from_active_frameworks(job_id=''):
-    active_framework_tasks = get_current_tasks(job_id)
+def get_running_tasks_from_active_frameworks(job_id='', include_orphans=True):
+    active_framework_tasks = get_current_tasks(job_id, include_orphanse=include_orphans)
```

Comment:
Yikes, thanks, I really shouldn't have rushed this so much...

I guess this implies poor unit tests too